README.md

```diff
@@ -132,10 +132,13 @@ Supported Scala versions: 2.13
 Running the `publishRelease.sh` script publishes a release of rawls-model, workbench-util and workbench-google to Artifactory.
 You should do this manually from the base directory of the repo when you change something in `model/src`, `util/src` or `google/src`.
 
-- The [`rawls-build` GitHub action](https://github.com/broadinstitute/terra-github-workflows/actions/workflows/rawls-build.yaml)
-publishes these libraries, but it makes "unofficial" `-SNAP` versions. This action runs on every dev build as part of the
-[`rawls-build-tag-publish-and-run-tests` workflow](https://github.com/broadinstitute/rawls/blob/develop/.github/workflows/rawls-build-tag-publish-and-run-tests.yaml).
-
+Note: We have started just using the automatically generated `-SNAP` versions published by [`rawls-build` GitHub action](https://github.com/broadinstitute/terra-github-workflows/actions/workflows/rawls-build.yaml) on every dev build. Here are detailed instructions for finding the name of the jar file:
+* Navigate to the [`rawls-build-tag-publish-and-run-tests` workflow](https://github.com/broadinstitute/rawls/blob/develop/.github/workflows/rawls-build-tag-publish-and-run-tests.yaml).
```

Comment:
Nit but I think the period here is a typo (or the "github action for your commit" part was meant to be deleted?)

Reply:
Fixed!